### ISSUE TITLE
Help with Koppen Geiger calculation

### ISSUE THREAD

**Issue by CatherineBuckland** [EvolEcolGroup/pastclim#61]
Hi pastclim team, 
I am new to the pastclim package, and keen to try and use it to calculate palaeo Koppen Geiger classifications using the package and data from the CHELSA-TraCE21K data repository. 

In the pastclim example, I see you use call in the data 'Temp', and 'Prec' as three dimentional arrays. I am interested in looking at particular periods, for example 5-4.5ka (as an example), is it simplest for me to produce new raster files for each monthly average, and then group them as a 12-layer raster stack, which I then call into the function. Would this work? 
Also I know that the precip value refers to averange monthly precipitation, but what metric is Temp? Is it average monthly temp, min or max? I cannot see Mean Monthly Temp available on CHELSA-TraCE21K - only max and min. 

Many thanks,

Catherine

**Comment by dramanica** [comment]
Yes, you can use the function on SpatRaster objects, so what you suggest should work. 
For mean temperature, you can simply use (tmin+tmax)/2; that's what was used in the original publication to compute bio1, which is the mean annual temperature.

**Comment by dramanica** [comment]
@CatherineBuckland is this issue resolved?

**Comment by dramanica** [comment]
I'll close this issue as it seems to be resolved. @CatherineBuckland let us know if you encounter further issues.
